.planemo.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+blah=`mktemp -d`
+/home/travis/build/fidelram/deepTools/foo/bin/planemo conda_init --conda_prefix $blah/conda
+export PATH=$blah/conda/bin:$PATH
+conda create -y --name deeptools_galaxy numpy matplotlib scipy
+source activate deeptools_galaxy
+conda install -c bioconda samtools
+git clone --depth 1 https://github.com/galaxyproject/galaxy.git clone
+cd clone
+./scripts/common_startup.sh --skip-venv --dev-wheels
+cd ..
+pip install . 
+/home/travis/build/fidelram/deepTools/foo/bin/planemo test --galaxy_root clone --test_data galaxy/wrapper/test-data/ --skip_venv \
+galaxy/wrapper/bamCompare.xml \
+galaxy/wrapper/bamCoverage.xml \
```

Comment:
What does the -r option do? One issue here is that we can't test all of the wrappers, since planemo doesn't seem to support custom data types (yet).